pkg/controller/oneagent/istio_test.go

```diff
@@ -19,34 +25,43 @@ var (
 	namespace  = "dynatrace"
 )
 
-func TestReconcileOneAgent_ReconcileIstio(t *testing.T) {
-	oa := newOneAgentSpec()
-	oa.ApiUrl = testAPIUrl
-	oa.Tokens = "token_test"
-	oa.EnableIstio = true
-	dynatracev1alpha1.SetDefaults_OneAgentSpec(oa)
+func TestReconcileOneAgent_CreateIstioObjects(t *testing.T) {
```

Comment:
Hmm, at the very least we aren't creating any objects here? At least not by the reconciler.